### DISCUSSION TITLE
How to install a specific version of Dask? Or downgrade to a previous version?

### DISCUSSION THREAD

**Discussion by LaurentEsingle** [dask/dask#7477] [Q&A] [answered]
I just rebuilt my application container, upgrading from dask 2021.2.0 to dask 2021.3.1

Now reading parquet files does not work anymore which is a serious issue.

I usually use:
```
pip install "dask[complete]"
```

I tried to install the previous version as follows:
```
pip install dask==2021.2.0
pip install "dask[array]"
pip install "dask[dataframe]"
pip install "dask[diagnostics]"
pip install "dask[distributed]"
```
But got the following error:
```
ERROR: distributed 2021.3.1 has requirement dask>=2021.03.0, but you'll have dask 2021.2.0 which is incompatible.
```

So my question is how do I solve this issue? It would be quicker for me to downgrade rather than finding a solution to the bug I am currently facing.

**Comment by jrbourbeau** [comment, marked as answer]
> Now reading parquet files does not work anymore which is a serious issue.

Sorry to hear that -- if you're able to open a bug report, that would be welcome

Something like:

```
pip install "dask[complete]==2021.2.0" distributed==2021.2.0
```

should allow you to downgrade to `2021.2.0` appropriately
